### PR TITLE
style and files added

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Add four components of P1 - typo, lists, modal and Navigation

### CODE REVIEW

File: Components/typography.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+:root {
+  --gray-text: gray;
```
Comment (deekshithmd) [inline Components/typography.css:2]: using color code will be good

File: Components/modal.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+.modal__border {
+  background-color: var(--rest-modal);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal {
+  width: 30%;
```
Comment (anshulraheja) [inline Components/modal.css:12]: I know you not focusing on building responsive components but a suggestion - the width of 30% will always depend upon the container. Instead, use max-width
Ex max-width: 300px

**Review by anshulraheja — APPROVED** [review]

File: Components/list.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  padding: 1.6rem;
+}
+
+.unordered__list {
+  justify-content: space-evenly;
+}
+
+.ordered__list {
+  justify-content: space-between;
+}
```
Comment (pankaj-prog) [inline Components/list.css:14]: you can improve on the naming, ordered list and unordered list are not list, they are containers containing the lists.

File: Components/navigation.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+.navigation__header2 {
+  padding: 1rem 2rem;
+  background-color: var(--primary-color);
+  align-items: center;
+  justify-content: space-between;
+}
+.company__name {
+  cursor: pointer;
+}
+.company__name2-nav > .navigation__links {
```
Comment (pankaj-prog) [inline Components/navigation.css:27]: You can name this header-title

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

File: Components/modal.css (hunk 2 above)
Comment (ashishkatiyar2404) [reply Components/modal.css:12]: ok, thanks Anshul it will help me while making it responsive. I will change % to px.

**Comment by ashishkatiyar2404** [comment]
I have made the changes.

**Pull request merged**